app/controllers/admin/user_creations_controller.rb

```diff
@@ -59,4 +59,38 @@ def destroy
       redirect_to works_path
     end
   end
+
+  def confirm_remove_pseud
+    authorize @work = Work.find(params[:id])
+
+    @orphan_pseuds = @work.orphan_pseuds
+    return unless @orphan_pseuds.empty?
+
+    flash[:error] = t(".must_have_orphan_pseuds")
+    redirect_to work_path(@work) and return
+  end
+
+  def remove_pseud
+    authorize @work = Work.find(params[:id])
+
+    pseuds = params[:pseuds]
+    orphan_account = User.orphan_account
+    if pseuds.blank?
+      pseuds = @work.orphan_pseuds
+      if pseuds.length > 1
+        flash[:error] = t(".must_select_pseud")
+        redirect_to work_path(@work) and return
+      end
+    else
+      pseuds = Pseud.find(pseuds).select { |p| p.user_id == orphan_account.id }
+    end
+
+    orphan_pseud = orphan_account.default_pseud
```

Comment:
You can inline this variable since it's only used once.

Reply:
It's used multiple times in the loop if multiple pseuds are removed. I want avoid the DB lookup/cache hit in the loop.

Reply:
Nice!